automated_analysis.py

```diff
@@ -464,5 +459,42 @@ def set_survey_percentages(survey_counts, total_survey_counts):
         fig.update_xaxes(tickangle=-60)
         fig.write_image(f"{automated_analysis_output_dir}/graphs/{plan.raw_field}_by_gender_normalised.png",
                         scale=IMG_SCALE_FACTOR)
+    
+    # Export safe to share raw messages for each episode to share with WUSC
+    log.info("Exporting safe to share raw messages for each episode...")
+    safe_to_share_messages = []  # of dict
+    for plan in PipelineConfiguration.RQA_CODING_PLANS[5:]: #loop through episode 6 -> because previous episodes had been manually processed
+        for cc in plan.coding_configurations:
+            code_to_messages = dict()
```

Comment:
You might like to use on ordereddict() here so that the codes get exported in the same order here as they do in the code scheme

Reply:
updated to use ordereddict() thanks.